bundles/org.openhab.binding.enocean/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -103,6 +103,66 @@
 		<state min="0" max="100" pattern="%.1f %%" readOnly="true"/>
 	</channel-type>
 
+	<channel-type id="co2">
+		<item-type>Number:Dimensionless</item-type>
+		<label>CO2</label>
+		<description>CO2 level in ppm</description>
+		<category>Carbondioxide</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>CO2</tag>
+		</tags>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="co">
+		<item-type>Number:Dimensionless</item-type>
+		<label>CO</label>
+		<description>CO level in ppm</description>
+		<category>Carbonmonoxide</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>CO</tag>
+		</tags>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="volatileOrganicCompounds">
+		<item-type>Number:Dimensionless</item-type>
+		<label>VOC</label>
+		<description>Volatile organic compoungs level in ppm/</description>
+		<category>Volatile organic compoungs</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>VOC</tag>
+		</tags>
+		<state pattern="%.2f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="volatileOrganicCompoundsId">
+		<item-type>String</item-type>
+		<label>VOC ID</label>
+		<description>Volatile organic compoungs identification</description>
+		<category>VOC</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>VOC</tag>
+		</tags>
+		<state pattern="%s" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="totalVolatileOrganicCompounds">
+		<item-type>Number:Dimensionless</item-type>
+		<label>TVOC</label>
+		<description>Total volatile organic compoungs level in ppb/</description>
+		<category>Total volatile organic compoungs</category>
```

Comment:
@jlaur I changed it to `<category>Volatile organic compounds</category>`. Not sure if this is a valid category, because I didn't manage to find a list with valid categories.

Reply:
The currently available categories/icons are here: https://www.openhab.org/docs/configuration/iconsets/classic/

Reply:
Thank you for the link, changed to `Gas`